database/console/main_test.go

```diff
@@ -11,7 +11,7 @@ import (
 var testDatabaseDocker *supportdocker.Database
 
 func TestMain(m *testing.M) {
-	if !env.IsWindows() {
+	if env.IsLinux() {
```

Comment:
Why change this?

Reply:
> Why change this?

GitHub's macOS envitonment not support running docker.